packages/keystone/src/lib/core/queries/output-field.ts

```diff
@@ -74,7 +74,7 @@ function getValueForDBField(
 }
 
 export function outputTypeField(
-  output: NextFieldType['output'],
+  output: Exclude<NextFieldType['output'], undefined>,
```

Comment:
This is required again right?
```suggestion
  output: NextFieldType['output'],
```